pkg/console/operator/operator.go

```diff
@@ -165,58 +171,64 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 		return err
 	}
 
-	if err := c.handleSync(operatorConfig, consoleConfig, infrastructureConfig); err != nil {
+	configs := configSet{
+		Console:        consoleConfig,
+		Operator:       operatorConfig,
+		Infrastructure: infrastructureConfig,
+	}
+
+	if err := c.handleSync(configs); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (c *consoleOperator) handleSync(operatorConfig *operatorsv1.Console, consoleConfig *configv1.Console, infrastructureConfig *configv1.Infrastructure) error {
-	operatorConfigCopy := operatorConfig.DeepCopy()
+func (c *consoleOperator) handleSync(configs configSet) error {
+	updatedStatus := configs.Operator.DeepCopy()
```

Comment:
not sure why `updatedStatus` ?

Reply:
ok, I see now after going though the whole PR, but still not sure about the name. Liked the original `operatorConfigCopy` more, since it was reflecting the object semantics... but opened for discussion

Reply:
Yeah, the tricky this is that it IS a `copy` of the whole `config`, but it is FOR status updates, only.  I think that what I'm aiming to express.

Reply:
Makes sense